lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/enumeration/PanStyles.java

```diff
@@ -3,20 +3,20 @@
 /**
  * PAN Style
  */
-public final class PanStyles {
-    private PanStyles(){
+@Deprecated public final class PanStyles {
+    @Deprecated private PanStyles(){
```

Comment:
@Deprecated is not good enough.
It should explain the reason deprecated and use what instead.